spec/features/audit/lists/filter_spec.rb

```diff
@@ -164,34 +164,47 @@
         end
 
         scenario "using autocomplete", js: true do
-          expect(page.current_url).not_to include("organisations%5B%5D=#{hmrc.content_id}")
```

Comment:
What about?

```ruby
context "filtering by organisation" do
  around(:each) do |example|
    Capybara.using_wait_time 5 do
      example.run
    end
  end
  #
end
```

Or, I use the scenario `metadata` data and configure it globally in `rails_helper.rb`?

Reply:
@pmanrubia done.